features/support/env.rb

```diff
@@ -1,4 +1,7 @@
 require 'cucumber/rails'
+require 'coveralls'
+Coveralls.wear!('rails')
```

Comment:
if you're going to merge coverage generated for both `rspec` and `cucumber` then you need to use...

```ruby
Coveralls. wear_merged!('rails')
```

... in both `env.rb` and `spec_helper.rb`
And you should do that at the top of the file. Before you require any other files